### PR TITLE
👨🏻‍💻 Improve documentation and test scripts

### PR DESCRIPTION
**Pull request by JuaniRios**
## What?
- Add comments to migration functinos
- Remove funding assets creation from chopsticks setup
- Fix paseo to use polimec instead of politest

### CODE REVIEW

**Comment by JuaniRios** [comment]
* **#332** <a href="https://app.graphite.dev/github/pr/Polimec/polimec-node/332?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @JuaniRios and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: nodes/parachain/src/chain_spec/polimec_paseo.rs [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 use sc_service::ChainType;
 
-use crate::chain_spec::{common::*, get_properties, Extensions, GenericChainSpec, DEFAULT_PARA_ID};
+use crate::chain_spec::{common::*, get_properties, Extensions, GenericChainSpec, DEFAULT_PARA_ID, polimec_rococo};
```
Comment (vstam1) [inline nodes/parachain/src/chain_spec/polimec_paseo.rs:23]: Why do we need polimec_rococo in this paseo file?

**Review by vstam1 — APPROVED** [review]
lgtm except 1 comment

File: pallets/funding/src/functions/7_ct_migration.rs [hunk 2]
```diff
@@ -24,6 +24,9 @@ impl<T: Config> Pallet<T> {
 		Ok(())
 	}
 
+	/// Handle the channel open request from the relay on behalf of a parachain.
+	/// If the para id belongs to a funded project with that id, then send an acceptance message and a request for a
```
Comment (lrazovic) [inline pallets/funding/src/functions/7_ct_migration.rs:28]:
```suggestion
	/// If the parachain id belongs to a funded project with the same project id, then send an acceptance message and a request for a
```

File: pallets/funding/src/functions/7_ct_migration.rs [hunk 3]
```diff
@@ -115,6 +118,8 @@ impl<T: Config> Pallet<T> {
 		}
 	}
 
+	/// Handle the channel accepted message of project->polimec from the relay on behalf of the project parachain.
```
Comment (lrazovic) [inline pallets/funding/src/functions/7_ct_migration.rs:121]:
```suggestion
	/// Handle the channel accepted message of project->Polimec from the relay on behalf of the project parachain.
```

File: pallets/funding/src/functions/7_ct_migration.rs [hunk 4]
```diff
@@ -326,6 +334,9 @@ impl<T: Config> Pallet<T> {
 		Ok(())
 	}
 
+	/// Migrate all the CTs of a project for a single participant
+	/// This entails transferring the funds from the polimec sovereign account to the participant account, and applying
```
Comment (lrazovic) [inline pallets/funding/src/functions/7_ct_migration.rs:338]:
```suggestion
	/// This entails transferring the funds from the Polimec sovereign account to the participant account, and applying
```

**Review by lrazovic — APPROVED** [review]
Just a few minor adjustments are needed. These suggestions are merely recommendations, so please feel free to use different wording.

**Pull request merged**